Author: Andreas Tille <tille@debian.org>
Last-Update: Wed, 20 Jul 2016 08:33:05 +0200
Description: Take over old patch to new upstream version.
 Include all optional libraries
Forwarded: not-needed

--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@
 
 # Set to 1 if you are using the random number generators in the GNU
 # Scientific Library.
-BUILD_GSL = 0
+BUILD_GSL = 1
 
 # Directory containing libgsl (used only when BUILD_GSL = 1).
 GSL_LIB_DIR = '/usr/lib'
@@ -26,7 +26,7 @@
 GSL_INC_DIR = '/usr/include/gsl'
 
 # Set to 1 if you are installing the fftw module.
-BUILD_FFTW = 0
+BUILD_FFTW = 1
 
 # Directory containing libfftw3 (used only when BUILD_FFTW = 1).
 FFTW_LIB_DIR = '/usr/lib'
@@ -35,7 +35,7 @@
 FFTW_INC_DIR = '/usr/include'
 
 # Set to 1 if you are installing the glpk module.
-BUILD_GLPK = 0
+BUILD_GLPK = 1
 
 # Directory containing libglpk (used only when BUILD_GLPK = 1).
 GLPK_LIB_DIR = '/usr/lib'
@@ -44,7 +44,7 @@
 GLPK_INC_DIR = '/usr/include'
 
 # Set to 1 if you are installing the DSDP module.
-BUILD_DSDP = 0
+BUILD_DSDP = 1
 
 # Directory containing libdsdp (used only when BUILD_DSDP = 1).
 DSDP_LIB_DIR = '/usr/lib'
